src/daemon/bitcoind/interface.rs

```diff
@@ -119,6 +119,10 @@ impl BitcoinD {
         })
     }
 
+    pub fn getmempoolentry(&self, txid: &Txid) -> Result<Json, BitcoindError> {
+        self.make_node_request("getmempoolentry", &params!(Json::String(txid.to_string())))
+    }
```

Comment:
This should be a bool method IMO, only returning true on entry here, false on `-5` error and error on other errors